### PR TITLE
fix Ubuntu 15.04 conditionnal

### PR DESCRIPTION
**Pull request by thiphariel**
Yup, I meant dependency _cough_

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by TeXitoi** [comment]
OK to test

File: build_navitia.sh [hunk 1]
```diff
@@ -179,7 +179,7 @@ then
     sudo pip install -r "$navitia_dir"/source/jormungandr/requirements.txt
     sudo pip install -r "$navitia_dir"/source/tyr/requirements.txt
     #we want a custom protobuff version
-    sudo pip install -U protobuf==2.5.0
+    sudo pip install -U protobuf==2.6.1
```
Comment (TeXitoi) [inline build_navitia.sh]: This line should just be removed, as it is managed by the requirements (as we want the distrib version)

Comment (thiphariel) [inline build_navitia.sh]: done :)

File: build_navitia.sh [hunk 2]
```diff
@@ -156,7 +156,7 @@ then
     version=`lsb_release -sr`
 
     # Fix Ubuntu 15.04 package
-    if [ "$distrib" = "Ubuntu" && "$version" = "15.04" ]; then
+    if [ "$distrib" = "Ubuntu" ] && [ "$version" = "15.04" ]; then
```
Comment (TeXitoi) [inline build_navitia.sh]: You can use `[ test -a test ]` instead of `[ test ] && [ test ]`

Comment (thiphariel) [inline build_navitia.sh]: Right. Will do that

**Pull request merged**